layouts/default.vue

```diff
@@ -228,7 +228,7 @@
               />
               <div class="profile-link">
                 <div class="username">@{{ $auth.user.username }}</div>
-                <div class="prompt">Go to my profile</div>
+                <div class="prompt">Go to your profile</div>
```

Comment:
just here too

Reply:
thank you, in our hubris we made the change on a phone web browser 😌